cli/tools/repl/editor.rs

```diff
@@ -304,7 +305,9 @@ impl Highlighter for EditorHelper {
   fn highlight<'l>(&self, line: &'l str, _: usize) -> Cow<'l, str> {
     let mut out_line = String::from(line);
 
-    for item in deno_ast::lex(line, deno_ast::MediaType::TypeScript) {
+    let mut lexed_items: VecDeque<_> =
+      deno_ast::lex(line, deno_ast::MediaType::TypeScript).into();
```

Comment:
Maybe using `peakable()` might be more appropriate (https://doc.rust-lang.org/std/iter/struct.Peekable.html)? Then you can use `lexed_items.next()` and `lexed_items.peek()`?

Reply:
done